core/src/test/java/org/elasticsearch/search/suggest/SuggestBuilderTests.java

```diff
@@ -39,13 +52,71 @@
 import org.elasticsearch.search.suggest.phrase.SmoothingModel;
 import org.elasticsearch.search.suggest.phrase.StupidBackoff;
 import org.elasticsearch.search.suggest.term.TermSuggestionBuilder;
+import org.elasticsearch.search.suggest.term.TermSuggestionBuilderTests;
+import org.elasticsearch.threadpool.ThreadPool;
+import org.elasticsearch.threadpool.ThreadPoolModule;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 
 import java.io.IOException;
 import java.util.Collections;
+import java.util.Locale;
 import java.util.Map.Entry;
 
+import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
+import static org.hamcrest.Matchers.equalTo;
+
 public class SuggestBuilderTests extends WritableTestCase<SuggestBuilder> {
 
+    private static Injector injector;
+    private static NamedWriteableRegistry namedWriteableRegistry;
+    private static IndicesQueriesRegistry queriesRegistry;
+    private static ParseFieldMatcher parseFieldMatcher;
+    private static Suggesters suggesters;
+
+    /**
+     * Setup for the whole base test class.
+     */
+    @BeforeClass
+    public static void init() throws IOException {
```

Comment:
I'm curious about why we need the whole injector setup here for this test? Was is needed for the xContent parsing in `testMalformedJsonRequestPayload()`? The parse context needed there should be easy to mock.